src/test/java/org/gridsuite/study/server/NetworkModificationTest.java

```diff
@@ -2429,8 +2418,8 @@ public void tearDown() throws IOException {
         TestUtils.assertQueuesEmptyThenClear(List.of(studyUpdateDestination), output);
 
         try {
-            // it returns an exception if a request was not matched by wireMock, but does not complain if it was not verified by 'verify'
```

Comment:
Why removing the comment ? If so, I think you should explain a bit more below

Reply:
Done